src/hooks/executor.rs

```diff
@@ -60,6 +60,11 @@ pub async fn executor_run(
             }
         });
 
+    if !is_duckdb_query && query.contains("time_bucket") {
```

Comment:
@rebasedming Would like your opinion on this? Should I be recreating time_bucket() as a native UDF in postgres or just error'ing out if its used on a PG table?